docs/release-notes/deprecate-modularity.rst

```diff
@@ -0,0 +1,12 @@
+:Type: Kickstart
+:Summary: Deprecate RPM modularity module
+
+:Description:
+    Based on the DNF team recommendation we decided to deprecate RPM modularity
+    feature in Anaconda. This is only a deprecation not removal but we will
+    reevaluate in future Fedora based on decisions from the DNF team.
```

Comment:
@KKoukiou it says 'Fedora' but I wanted to keep this in sync with master. WDYT?

Reply:
We don't use the release-notes directory in RHEL branches. If you cherry-picked the commit from master, just leave it here this file as well, but RHEL 10 deprecations are handled in external document.